proposals/lip-0053.md

```diff
@@ -593,6 +593,10 @@ def execute(trs: Transaction) -> None:
     # Update the context to indicate that now we stop the CCM processing.
     ctx.ccmProcessing = False
 
+    # Set the transaction ID back as default topic.
+    # The default topic is added automatically by emitEvent.
+    eventsDefaultTopic = transaction ID
```

Comment:
1. I would do this for LIP 0054 as well.
2. Maybe it's a bit cleaner to have than only a single `eventsDefaultTopic = ccmID` statement before the first for loop. But it's also ok to have like this if you prefer.

Reply:
ah, forget the 2nd point. I just read your other comment.

Reply:
In LIP54 there are no other events emitted in the transaction execution, that's why I didn't add it

Reply:
ok, I see.